packages/popover/src/Popover.stories.tsx

```diff
@@ -111,23 +118,28 @@ const meta: StoryMetaType<typeof Popover> = {
       },
       // eslint-disable-next-line react/display-name
       decorator: Instance => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        const [active, setActive] = useState<boolean>(false);
```

Comment:
how is this working?

Reply:
Oh I see this is just the generated decorator?
can we do 
```
const [_, setActive] = useState<boolean>(false);
```

or just cast a void function as a click handler to Button?